arithmetic_analysis/jacobi_iteration_method.py

```diff
@@ -144,20 +144,12 @@ def strictly_diagonally_dominant(table: NDArray[float64]) -> bool:
 
     rows, cols = table.shape
 
-    is_diagonally_dominant = True
-
-    for i in range(0, rows):
-        total = 0
-        for j in range(0, cols - 1):
-            if i == j:
-                continue
-            else:
-                total += table[i][j]
-
+    for i in range(rows):
+        total = sum(table[i][j] for j in range(cols - 1) if i != j)
         if table[i][i] <= total:
             raise ValueError("Coefficient matrix is not strictly diagonally dominant")

```

Comment:
```suggestion
            return False
```
The function currently returns `True` if `table` is strictly diagonally dominant, but it raises a `ValueError` if `table` isn't. It makes more sense to just have it return `False` instead.